Update icons reference in doc metadata docs (#16584)
pypi/warehouse: docs/user/project_metadata.md

indicate a prefix. It means that any name starting with the specified pattern 
will be recognized.

#### General URL

| Icon                                      | Name             | Description                     | Aliases                                                                                                  |
|:------------------------------------------|:-----------------|:--------------------------------|:---------------------------------------------------------------------------------------------------------|
| :fontawesome-solid-house-chimney:         | Homepage         | For the project homepage        |                                                                                                          |
| :fontawesome-solid-cloud:                 | Download         | A download link                 |                                                                                                          |
| :fontawesome-solid-scroll:                | Changelog        | Changelog information           | Change log, Changes, Release notes, News, What's new, History                                            |

> | :fontawesome-solid-book:                  | Documentation^*^ | Project <br/><br/>documentation | Docs^*^ , an URL pointing on [Read the Docs]: domains or an URL starting by `docs.` or `documentation.`. |

Reviewer: ```suggestion
| :fontawesome-solid-book:                  | Documentation^*^ | Project <br/><br/>documentation | Docs^*^ , an URL pointing to [Read the Docs] domains or an URL starting with `docs.` or `documentation.` |
```

> | :fontawesome-solid-book:                  | Documentation^*^ | Project documentation | Docs^*^ , an URL pointing to [Read the Docs] domains or an URL starting with `docs.` or `documentation.` |

Reviewer 2: There's a lot of these where we match based on the URL instead of the name that aren't documented. I think these should probably get all get documented in the "Hosting Platforms"/"Social Media Platforms" sections instead, with a "URL" column, not just RTD.

Reviewer 2: Also just noticed a typo in the heading: "Social Medias Platforms"

Author: Good catch!

> | :fontawesome-solid-book:                  | Documentation^*^ | Project <br/><br/>documentation | Docs^*^ , an URL pointing to [Read the Docs] domains or an URL starting with `docs.` or `documentation.` |

Reviewer 2: ```suggestion
| :fontawesome-solid-book:                  | Documentation^*^ | Project documentation | Docs^*^ , an URL pointing to [Read the Docs] domains or an URL starting with `docs.` or `documentation.` |
```

What's with the line breaks here? This renders somewhat weird: https://docspypiorg--16584.org.readthedocs.build/project_metadata/

Reviewer 3: Removing!
